Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.CheckSum.Tools/Classes/HashingHelpers.cs

```diff
@@ -75,7 +75,7 @@ public static string BuildSHA512HashString(byte[] hashBytes)
 
         public static string BuildRIPEMD160HashString(byte[] hashBytes)
         {
-            StringBuilder builder = new StringBuilder();
+            StringBuilder builder = new StringBuilder(40);
```

Comment:
Comment / State why "40"

Reply:
Will do, it's basically the required length of the string